jwst/coron/median_replace_img.py

```diff
@@ -47,7 +48,7 @@ def median_fill_value(input_array, input_dq_array, bsize, bad_bitvalue, xc, yc):
         dq_array = input_dq_array[xc - hbox:xc + hbox + 1, yc - hbox: yc + hbox + 1]
     except IndexError:
         # If the box is outside the data return 0
-        log.warning('Box for median filter is outside the data.')
+        # log.warning('Box for median filter is outside the data.')
```

Comment:
Should reinstate this warning for the final version. Or does it get triggered a lot for some kinds of data?

Reply:
Oops, you're correct. That's not the warning I'm worried about. I must have commented it out by accident. Thanks very much for catching it.